submissions/f3d0t/friends-app/app.js

```diff
@@ -0,0 +1,166 @@
+const MAIN = document.querySelector(".main");
+const FILTERS = document.querySelector(".filters");
+const RESET_BUTTON = document.querySelector(".filters__button");
+const DAY_NIGHT = document.querySelector(".day_night");
+const FRIENDS_COUNT = 60;
+const REQUEST_LINK = `https://randomuser.me/api/?results=${FRIENDS_COUNT}`;
+
+const fetchData = async (requestLink) => {
+	try {
+		const response = await fetch(requestLink);
```

Comment:
https://www.tjvantoll.com/2015/09/13/fetch-and-errors/

Reply:
should I use pure promises syntax without await?

Reply:
No. But you should handle response status